src/__tests__/actions/tagsActions.test.js

```diff
@@ -0,0 +1,52 @@
+import configureStore from 'redux-mock-store'
+import * as tagsActions from '../../redux/actions/tagsActions'
+import { ADD_TAG, EDIT_TAG, DELETE_TAG, } from '../../constants'
+
+const mockStore = configureStore()
+const store = mockStore()
+
+describe('tagsActions', () => {
+  const prevName = 'myTag'
+  const newName = 'MyTAG'
+  const tagName = 'myTag'
+
+  beforeEach(() => {
+    store.clearActions()
+  })
+
+  test('should dispatch the addTag action', () => {
+    const expectedAction = [
```

Comment:
`expectedActions`

Reply:
No, a mean one action, because an action creator generate only one action

Reply:
@Dekim12 but you define it as an array